### ISSUE TITLE
Show device dashboard link in `resin device <uuid>`

### ISSUE THREAD

**Issue by lekkas** [resin-io/resin-cli#395]
Hey @jviotti , @imrehg , @shaunmulligan 

What are your thoughts on adding the dashboard link of a device in the output of `resin device <uuid>` command, so that we can have something like:

``` bash
~# resin device d6a0c17
ID:                 47909
DEVICE TYPE:        parallella
STATUS:             offline
[...]
OS VERSION:         Resin OS 1.12.0
DASHBOARD LINK:     https://dashboard.resin.io/apps/13675/devices/47909/summary
```

This will provide a fast way get the dashboard link from a device when a user (or admin) only has a uuid.

**Comment by jviotti** [comment]
Looks like a great idea. Is there a way we can get the link without
hardcoding its structure in the CLI?

On Fri, Sep 23, 2016 at 01:40:53AM -0700, Kostas Lekkas wrote:
> Hey @jviotti , @imrehg , @shaunmulligan 
> 
> What are your thoughts on adding the dashboard link of a device in the output of `resin device <uuid>` command, so that we can have something like:
> 
> ```bash
> ~# resin device d6a0c17
> ID:                 47909
> DEVICE TYPE:        parallella
> STATUS:             offline
> [...]
> OS VERSION:         Resin OS 1.12.0
> DASHBOARD LINK:     https://dashboard.resin.io/apps/13675/devices/47909/summary
> ```
> 
> This will give both users and admins to easily get the dashboard link from a device when they only have a uuid.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/resin-io/resin-cli/issues/395

-- 
Juan Cruz Viotti
Software Engineer

**Comment by lekkas** [comment]
Cool, I'll send a PR!

> Is there a way we can get the link without
hardcoding its structure in the CLI?

I'm not sure, @emirotin and @dfunckt might know more about this.

**Comment by imrehg** [comment]
I wonder how much hard-coding it needs, since the cli needs to work with `RESINRC_RESIN_URL`, which already includes just the main domain (i.e. "dashboard" is already hard-coded somewhere, since it's used with `resin login` too). A quick scan of the API calls available don't turn up info like that full link.

**Comment by emirotin** [comment]
There's no easy way to build the UI URLs, so your best bet is to isolate all such manipulations in a single module and if eventually things change it will be easy to trace them and fix.

**Comment by lekkas** [comment]
Fixed by https://github.com/resin-io/resin-cli/pull/398 (landed in `resin-cli@5.10.0`)
